Services/UI/OJS.Services.Ui.Business/Implementations/HangfireBackgroundJobsBusinessService.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace OJS.Services.Ui.Business.Implementations;
+
+using OJS.Services.Common.Data;
+using System.Threading.Tasks;
+using OJS.Services.Common;
+
+public class HangfireBackgroundJobsBusinessService : IHangfireBackgroundJobsBusinessService
+{
+    private readonly ISubmissionsForProcessingCommonDataService submissionsForProcessingData;
+    private readonly IUserProviderService userProviderService;
+
+    public HangfireBackgroundJobsBusinessService(
+        ISubmissionsForProcessingCommonDataService submissionsForProcessingData,
+        IUserProviderService userProviderService)
+    {
+        this.submissionsForProcessingData = submissionsForProcessingData;
+        this.userProviderService = userProviderService;
+    }
+
+    public Task<int> GetUnprocessedTotalCountJob()
```

Comment:
If this is not part of the interface, should not be here as well

Reply:
Removed